recipe/meta.yaml

```diff
@@ -20,7 +19,7 @@ build:
   - vc14  # [win and py35]
 
 # https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/issues/29
-  skip: true  # [win and py35]
+  #skip: true  # [win and py35]
```

Comment:
I suspect the new release fixed the compilation error with Python 3.5 on Windows